Tensile/Contractions.py

```diff
@@ -316,8 +316,9 @@ def CompoundPredicates(cls, state, problemType):
         if "KernelLanguage" in state:
             rv += [cls("KernelLanguageCompatible", value=state["KernelLanguage"])]
 
-        if 'GlobalSplitU' in state and state['GlobalSplitU'] > 1:
-            rv += [cls("DeterministicMode", value = False)]
+        if ('GlobalSplitU' in state) and (state['GlobalSplitU'] > 1):
```

Comment:
Why use _GlobalAccumulation instead of GlobalSplitUAlgorithm?

Reply:
GlobalSplitUAlgorithm is used to decide which algorithm you want to use if GSU is not 0.
_GlobalAccumulation is used to decide do we have to do accumulation in workspace.
GlobalSplitUAlgorithm is a user define value, _GlobalAccumulation is a derived value.
I don't want to modify GlobalSplitUAlgorithm value by Tensile logic.

state["_GlobalAccumulation"] = 0
    if ((dataType.isBFloat16() or dataType.isHalf())
      and state["ProblemType"]["HighPrecisionAccumulate"] \
      and state["GlobalSplitU"] > 1 \
      and state["EnableMatrixInstruction"]):
      state["_GlobalAccumulation"] = state["GlobalSplitUAlgorithm"]